frontend/src/components/reviews-replies/ReviewCard.vue

```diff
@@ -35,5 +35,5 @@ export default {
 }
 </script>
 
-<style scoped src='../../css/card.less'>
+<style scoped src='../../css/card.less' lang='less'>
```

Comment:
it's weird how less is called less when it adds more to css

Reply:
🥇